### PR TITLE
Added Poppins font and design system colors

### PR DESCRIPTION
**Pull request by kelsiliul**
Add UI elements.

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+Score
```
Comment (thisjustin123) [inline .idea/.name:1]: `.idea/` files shouldn't be tracked; they should be included in the `.gitignore`. Here's a good template for that:

```
# Gradle files
.gradle/
build/

# Local configuration file (sdk path, etc)
local.properties

# Log/OS Files
*.log

# Android Studio generated files and folders
captures/
.externalNativeBuild/
.cxx/
*.apk
output.json

# IntelliJ
*.iml
.idea/
misc.xml
deploymentTargetDropDown.xml
render.experimental.xml

# Keystore files
*.jks
*.keystore

# Google Services (e.g. APIs or Firebase)
app/google-services.json

# Android Profiling
*.hprof

# Secrets
secrets.properties
```

**Review by thisjustin123 — CHANGES_REQUESTED** [review]
Please edit the gitignore and git remove the .idea files from the PR!

**Review by kelsiliul — COMMENTED** [review]
Updated .gitignore

**Review by thisjustin123 — CHANGES_REQUESTED** [review]
I don't see the `.gitignore` changed here? I just see that you git removed the `.idea/` files.

Make sure to actually modify & push the changes to the `.gitignore` file. I gave a good model for what the `.gitignore` file should look like in my previous review.

**Comment by kelsiliul** [comment]
Updated .gitignore

**Review by thisjustin123 — APPROVED** [review]
Noice 😎

**Pull request merged**